Blaise-Shyaka/Politico
# [#171638354] Create page for admin to view all parties

# What does this PR do?

It creates a page that allows the admin to view all existing political parties

# Description of tasks to be completed

- Create a page displaying the logo, name, and headquarters of each party
- Make the page responsive at least on mobile, tablet and desktop

# How should this be manually tested?

- Clone repository and `cd /templates/html/`
- Open `admin-view-parties.html`
- Check the presence of a menu
- Check whether the logo, name, and headquarter of each party are displayed
- Resize the browser to verify the page responsiveness

# What are the relevant pivotal tracker stories?

#171638354

# Screenshots

![Screenshot (36)](https://user-images.githubusercontent.com/52489421/75936183-1d66de00-5e8a-11ea-8baa-3ce78404717b.png)
![Screenshot (38)](https://user-images.githubusercontent.com/52489421/75936212-2f488100-5e8a-11ea-9cbe-3f9c17292649.png)
![Screenshot (39)](https://user-images.githubusercontent.com/52489421/75936336-7b93c100-5e8a-11ea-8183-b9ef038be09a.png)

![Screenshot (37)](https://user-images.githubusercontent.com/52489421/75936194-23f55580-5e8a-11ea-8c24-4fbb85b0e310.png)
